rfcs/RFC-Uploads.md

```diff
@@ -0,0 +1,404 @@
+# User Uploads
+
+## For the cellxgene Data Portal
+
+**Authors:** [Trent Smith](mailto:trent.smith@chanzuckerberg.com)
+
+**Approvers:** [Arathi Mani](mailto:arathi.mani@chanzuckerberg.com), [Timmy Huang](mailto:thuang@chanzuckerberg.com),
+[Marcus Kinsella](mailto:mkinsella@chanzuckerberg.com), [Eduardo Lopez](mailto:elopez@chanzuckerberg.com), [Brian Raymore](mailto:braymor@chanzuckerberg.com)
+[Sara Gerber](mailto:sara.gerber@chanzuckerberg.com)
+
+## tl;dr
+
+A user can upload files of any size to Data Portal using either a shared link from their cloud provider or directly from their browser.
+
+## Glossary
+
+- CSP(Cloud Storage Provider) - Cloud storage services such as Dropbox or Google Drive.
```

Comment:
Note that CSP also means "content security policy" and could be potentially confusing to readers (it confused me a couple times already).

Reply:
I understand, which is why I defined at the top and did not reference "content security policy" in the document. If you come up with something better, I'm all ears.

Reply:
Don't rely on acronyms because they're ambiguous?